src/components/sbb-card/sbb-card.scss

```diff
@@ -73,35 +73,38 @@
   &:focus-visible {
     @include focusOutline;
   }
-}
 
-:host([active]) {
-  .sbb-card::before {
-    border: var(--sbb-border-width-2x) solid var(--sbb-color-smoke-default);
+  :host([href]) & {
```

Comment:
Please add `{ reflect: true}` to the href Prop in sbb-card.tsx (this might break otherwise if the property is changed programmatically)